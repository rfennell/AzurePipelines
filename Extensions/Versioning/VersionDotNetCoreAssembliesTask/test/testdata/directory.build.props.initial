<Project>
  <PropertyGroup>
    <Version>1.2</Version>
    <FileVersion>1.2.1811.0</FileVersion>
    <Product>A test</Product>
    <Company>ABC Inc</Company>
    <Copyright>Copyright Â© </Copyright>
    <NeutralLanguage>en</NeutralLanguage>
  </PropertyGroup>
</Project>