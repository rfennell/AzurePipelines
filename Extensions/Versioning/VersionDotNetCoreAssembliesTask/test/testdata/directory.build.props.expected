<Project>
  <PropertyGroup>
    <Version>9.9.9.9</Version>
    <FileVersion>1.2.1811.0</FileVersion>
    <Product>A test</Product>
    <Company>ABC Inc</Company>
    <Copyright>Copyright Â© </Copyright>
    <NeutralLanguage>en</NeutralLanguage>
  </PropertyGroup>
</Project>